browser/store/stats.js

```diff
@@ -13,24 +16,81 @@ const statsObj = {}
 /**
  * ACTION CREATORS
  */
-const getStats = stats => ({type: GET_STATS, stats})
+const getStats = stats => ({ type: GET_STATS, stats });
+
+const getMostWins = winner => ({ type: GET_MOST_WINS, winner });
+
+const getMostKills = kills => ({ type: GET_MOST_KILLS, kills });
+
+const getMostPlays = plays => ({ type: GET_MOST_PLAYS, plays });
 
 
 export const getStatsThunk = () =>
-  dispatch =>
+    dispatch =>
     axios.get('/api/stats')
-      .then(res => {
+    .then(res => {
         console.log('this was called', res.data);
         return dispatch(getStats(res.data))
-      })
-      .catch(err => console.log(err))
-
-
-export default function (state = [], action) {
-  switch (action.type) {
-    case GET_STATS:
-      return action.stats
-    default:
-      return state
-  }
-}
+    })
+    .catch(err => console.log(err))
+
+export const getMostWinsThunk = () =>
```

Comment:
might want to added a couple comments documenting how we are planning on changing this.